### PR TITLE
Update gdal and rebuild revdeps

### PR DESCRIPTION
**Pull request by malfisya**
**Summary**

- Update gdal to 3.8.3
- Update fgt to 0.4.10
- Update cpd to 0.5.4
- Update pdal to 2.5.6
- Rebuild : flightgear, simgear, qmapshack , qgis, and vtk
- Resolves https://github.com/getsolus/packages/issues/1067

### CODE REVIEW

**Comment by ReillyBrogan** [comment]
Legend

**Comment by ReillyBrogan** [comment]
Marking as draft since PDAL rebuilds are not done.

File: packages/f/fgt/files/build-shared-lib.patch [hunk 1]
```diff
@@ -21,15 +30,18 @@
...
      )
  
-@@ -162,9 +164,9 @@ write_basic_package_version_file(
+@@ -173,9 +175,9 @@ write_basic_package_version_file(
  install(FILES
      ${PROJECT_BINARY_DIR}/fgt-config.cmake
      ${PROJECT_BINARY_DIR}/fgt-config-version.cmake
 -    DESTINATION lib/cmake/fgt
-+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/fgt
++    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmkae/fgt
```
Comment (Staudey) [inline packages/f/fgt/files/build-shared-lib.patch:38]:
```suggestion
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/fgt
```

File: packages/f/fgt/files/build-shared-lib.patch [hunk 2]
```diff
@@ -21,15 +30,18 @@
...
      )
  install(EXPORT fgt-targets
 -    DESTINATION lib/cmake/fgt
-+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/fgt
++    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmkae/fgt
```
Comment (Staudey) [inline packages/f/fgt/files/build-shared-lib.patch:42]:
```suggestion
+    DESTINATION ${CMAKE_INSTALL_LIBDIR}/cmake/fgt
```

File: packages/f/fgt/pspec_x86_64.xml [hunk 3]
```diff
@@ -22,8 +22,12 @@ The Gauss transform is a common operation that computes the per-point similarity
 </Description>
         <PartOf>programming.library</PartOf>
         <Files>
+            <Path fileType="library">/usr/lib64/cmkae/fgt/fgt-config-version.cmake</Path>
+            <Path fileType="library">/usr/lib64/cmkae/fgt/fgt-config.cmake</Path>
+            <Path fileType="library">/usr/lib64/cmkae/fgt/fgt-targets-relwithdebinfo.cmake</Path>
+            <Path fileType="library">/usr/lib64/cmkae/fgt/fgt-targets.cmake</Path>
```
Comment (Staudey) [inline packages/f/fgt/pspec_x86_64.xml:28]: See above for why this happens

**Review by Staudey — COMMENTED** [review]
I know this has been marked as a draft now, but just happened to spot this while quickly scanning the changes.

**Comment by ReillyBrogan** [comment]
> I know this has been marked as a draft now, but just happened to spot this while quickly scanning the changes.

Being marked as a draft doesn't mean it can't receive review comments. It just means that it can't be merged.

**Review by ReillyBrogan — APPROVED** [review]

**Pull request merged**